server/lib/weather/weather.command.js

```diff
@@ -11,19 +37,55 @@ const { ServiceNotConfiguredError, NoValuesFoundError } = require('../../utils/c
  */
 async function command(message, classification, context) {
   let weather;
+  let weatherDay;
   try {
     const houses = await this.house.get();
     const house = houses[0];
     if (!house || !house.latitude || !house.longitude) {
       throw new NoValuesFoundError();
     }
+    weather = await this.get(house);
     switch (classification.intent) {
       case 'weather.get':
-        weather = await this.get(house);
         context.temperature = weather.temperature;
         context.units = weather.units === 'metric' ? '°C' : '°F';
+
         await this.messageManager.replyByIntent(message, `weather.get.success.${weather.weather}`, context);
         break;
+      case 'weather.tomorrow':
```

Comment:
I know it's not what we did in the current implement, but could you require the constant file and use the constant here instead of the string? This will avoid errors :)

Reply:
Done